src/Helpers/JWKFetcher.php

```diff
@@ -17,19 +17,34 @@
 class JWKFetcher
 {
     /**
-     * How long should the cache persist? Set to 10 minutes.
+     * Default length of cache persistence. Defaults to 10 minutes.
      *
      * @see https://www.php-fig.org/psr/psr-16/#12-definitions
      */
     const CACHE_TTL = 600;
 
+    /**
+     * How long should the cache persist? Defaults to value of CACHE_TTL.
+     *
+     * @see https://www.php-fig.org/psr/psr-16/#12-definitions
+     */
+    private $ttl = 600;
```

Comment:
why not set the value to the default/const? That way if the default changes this value will also get changed.

Reply:
I was just assigning it in the constructor, but you're right, that's a cleaner way for doing things. Updating now!